sites/hashdev/package.json

```diff
@@ -27,12 +27,14 @@
     "@mui/system": "5.5.1",
     "@types/md5": "2.3.2",
     "axios": "0.26.1",
+    "calculation-block": "0.2.2",
```

Comment:
@thehabbos007 Do we want to do this vs import from the BP Hub? cc @CiaranMn

Reply:
Yes I think we do, but am not sure what the mechanism to do that is for mock-block-dock (i don't think it can right now?).
We can always change this, though, which is good.

Reply:
A future `mock-block-dock-block`?